content/en/docs/reference/kubernetes-api/workload-resources/pod-v1.md

```diff
@@ -1592,7 +1592,16 @@ LifecycleHandler defines a specific action that should be taken in a lifecycle h
 
     Optional: Host name to connect to, defaults to the pod IP.
 
+- **sleep** (SleepAction)
```

Comment:
Don't touch this file yet.
This file is auto-generated and will be refreshed once 1.29 is out of the door.

Reply:
Okay, how is this file generated? Should I pay attention to it?

Reply:
No need to pay attention to it at the moment.
v1.29 website (including your PR) won't get published until 1.29 becomes a thing.